everest.yaml

```
@@ -1,5 +1,5 @@
 - Name: CommunalHelper
-  Version: 0.0.0-dev
+  Version: 1.19.4
```

don't change this, it's intentionally left there so that the build hook on GitHub replaces it with the latest version upon release
```suggestion
  Version: 0.0.0-dev
```